static/app/views/replays/detail/layout/focusArea.tsx

```diff
@@ -39,8 +43,17 @@ function FocusArea({}: Props) {
         </MemoryTabWrapper>
       );
     case TabKey.CONSOLE:
-    default: {
       return <Console />;
+    case TabKey.TAGS:
+      return <TagPanel />;
+    case TabKey.BREADCRUMBS:
+    default: {
+      return (
+        <Breadcrumbs
+          frames={replay?.getChapterFrames()}
+          startTimestampMs={replay?.getReplay()?.started_at?.getTime() || 0}
```

Comment:
Looks like other views access the replay via the hook instead of through props, we should do the same here

Reply:
idk if it's a huge deal tbh. the idea is to narrow the scope of re-renders, but i didn't strongly test that it's better. 

lets ship it as-is and we can profile and revisit this again some other time.